### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by nem035**
I thought about adding a foot note on "block-scoped" on line 58 but it's mentioned in the insight right before this so I'll leave it to you guys @kapnobatai137 and @Stefan-Stojanovic to decide 🤷‍♂

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/ecmascript-2015/intro-ecmascript/let-part-2.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/ecmascript-2015/intro-ecmascript/let-part-2.md?ref=e84c30b0cb4cf2b91cfaf44dcfc0d973a7e41271)

**Comment by kapnobatai137** [comment]
added this footnote:

By this we mean that any variable defined with `let` (or `const`) is only accessible in its scope. This is best shown through an example:

```js
if (true) {
  let foo = 1;
}
console.log(foo);
// undefined
```

Here, `foo` is only accessible in the `if` scope.

@nem035 let me know what you think

**Comment by nem035** [comment]
@kapnobatai137 I like it 👍

File: javascript/ecmascript-2015/intro-ecmascript/let-part-2.md [hunk 1]
```diff
@@ -126,3 +122,19 @@ for (func of letFuncs) {
 * [4, 4, 4]
 * [2, 2, 2]
 * [1, 2, 3]
+
+---
+## Footnotes
+
+[1:Block-scope]
+By this we mean that any variable defined with `let` (or `const`) is only accessible in its scope. This is best shown through an example:
```
Comment (nem035) [inline javascript/ecmascript-2015/intro-ecmascript/let-part-2.md]:
```suggestion
By this we mean that any variable defined with `let` (or `const`) is only accessible in its scope (between the nearest pair of `{}`). This is best shown through an example:
```

File: javascript/ecmascript-2015/intro-ecmascript/let-part-2.md [hunk 2]
```diff
@@ -1,21 +1,13 @@
...
 aspects:
   - introduction
 
-inAlgoPool: false
-
-standards:
-  javascript.new-declaration-keywords.0: 10
-
 links:
   - '[MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let){website}'
```
Comment (kapnobatai137) [inline javascript/ecmascript-2015/intro-ecmascript/let-part-2.md]:
```suggestion
  - '[MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let){website}'
```

**Pull request merged**